crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -1047,6 +1049,17 @@ impl CardData for Card {
                 .to_string(),
         ))
     }
+    fn get_card_expiry_month_2_digit(&self) -> Result<Secret<String>, errors::ConnectorError> {
+        let month = self.card_exp_month.clone().expose();
+        let month_length = month.len();
+        match month_length {
+            2 => Ok(self.card_exp_month.clone()),
+            1 => Ok(Secret::new(format!("0{}", month))),
```

Comment:
Would have preferred using the `CardExpirationMonth` type (in these structs) instead of a string, it has implementations for this already.

https://github.com/juspay/hyperswitch/blob/2f55be45fba43484b5d1ae36864028388b02e67a/crates/cards/src/lib.rs#L41C6-L45